melted_UKBB_extract.py

```diff
@@ -3,20 +3,22 @@
 import logging
 import sys
 
-from config import Config
+from typing import Optional
 
+import polars as pl
+import pathlib as p
+
+from config import Config, load_config
 
 def extract_UKBB_tabular_data(
     config: Config,
-    data_file: str | None = None,
-    dictionary_file: str | None = None,
-    coding_file: str | None = None,
-    verbose: str | None = False,
-):
-    import polars as pl
-    import pathlib as p
+    data_file: str,
+    dictionary_file: str,
+    coding_file: str,
+    verbose: Optional[bool] = False,
+) -> tuple[pl.DataFrame, pl.DataFrame | None, pl.DataFrame, pl.DataFrame]:
 
-    pl.Config.set_verbose(verbose)
+    pl.Config.set_verbose(verbose == True)
```

Comment:
why this change? can verbose not be passed directly into this?

Reply:
According to MyPy, this method only accepts `bool`. It is possible `None` would work at runtime, but why not be safe and make sure it is a `bool`.

Reply:
why would it ever be none? the function call had  a default value of  `verbose = False`

Reply:
yup, this is gone now.

Reply:
1a16e48edd1db5c39e8a6b909a7502861abb8cfe